extensions/cornerstone/src/commandsModule.ts

```diff
@@ -430,6 +430,37 @@ function commandsModule({
 
       actions.setViewportWindowLevel({ ...props, viewportId });
     },
+    setWindowLevelPreset: ({ presetName, presetIndex }) => {
```

Comment:
Having these as commands is really nice for allowing different bindings.

Reply:
Thanks